Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.yml

```diff
@@ -0,0 +1,90 @@
+category: Utilities
+commonfields:
+  id: XSOAR File Management
+  version: -1
+configuration:
+- additionalinfo: Make sur XSOAR config 'External Host Name' is set and let this field empty otherwise set the external ip of XSOAR. Using https://127.0.0.1 don't work.
```

Comment:
```suggestion
- additionalinfo: Make sure XSOAR config 'External Host Name' is set and let this field empty otherwise set the external ip of XSOAR. Using https://127.0.0.1 don't work.
```